src/Alert.php

```diff
@@ -60,14 +62,14 @@ public function addClass(string $value): self
      *
      * @param array $values Attribute values indexed by attribute names.
      *
-     * @return self A new instance of the current class with the specified attributes.
+     * @return self A new instance with the specified attributes.
      *
      * @see {\Yiisoft\Html\Html::renderTagAttributes()} for details on how attributes are being rendered.
      */
     public function attributes(array $values): self
     {
         $new = clone $this;
-        $new->attributes = $values;
+        $new->attributes = array_merge($new->attributes, $values);
```

Comment:
`attributes` should replace value, `addAttributes` — merge with current value

Reply:
add method `addAttributes()`.

`attributes()`  for replace "attributes" property
`addAttributes()` for merge with current value.

Example: https://github.com/yiisoft/form/blob/d84e1200ad5478c6cc20581773b43a0c13d921d7/src/Field/Part/Hint.php#L47-L59

Reply:
done.